### PR TITLE
Fix input type=text selection reset.

### PR DESCRIPTION
**Pull request by jbucaran**
ᕕ( ᐛ )ᕗ 🎉

Related #137.
Ping @mrkstwrt.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -145,11 +145,13 @@ export default function(app) {
   function updateElementData(element, oldData, data) {
     for (var name in merge(oldData, data)) {
       var value = data[name]
-      var oldValue = oldData[name]
+      var oldValue = name === "value" || name === "checked"
```
Comment (dodekeract) [inline src/app.js]: Why check for `checked` and `value`? Any explanation?

Comment (jbucaran) [inline src/app.js]: These are the two attributes whose actual value change when you interact with the component. So, when diffing an element's data, we want to look at the real DOM (not the stuff in oldData).

**Comment by zaceno** [comment]
@jbucaran -- will this fix be coming to picodom as well (or is it not necessary?)

**Comment by jbucaran** [comment]
@zaceno Yeah, already fixed in picodom. https://github.com/picodom/picodom/issues/6

**Pull request merged**